CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 - [#4721](https://github.com/influxdb/influxdb/pull/4721): Export tsdb.InterfaceValues
 - [#4681](https://github.com/influxdb/influxdb/pull/4681): Increase default buffer size for collectd and graphite listeners
 - [#4659](https://github.com/influxdb/influxdb/pull/4659): Support IF EXISTS for DROP DATABASE
+- [#4685](https://github.com/influxdb/influxdb/pull/4685): Heal Raft Cluster
```

Comment:
This only promotes a raft node when it sees a gap in the peers.  It can only see a gap in the peers if someone issues a `DROP SERVER`.

They type of healing you are referring too is beyond the scope of this PR, and I'm not sure we would actually want to take those steps or not.